src/reference/asciidoc/redis.adoc

```diff
@@ -791,12 +791,104 @@ You should set the expiry at a large enough value to prevent this condition, but
 Starting with version 5.0, the `RedisLockRegistry` implements `ExpirableLockRegistry`, which removes locks last acquired more than `age` ago and that are not currently locked.
 
 
+[[redis-stream-outbound]]
+=== Redis Stream Outbound Channel Adapter
+
+Spring Integration 5.4 introduced Reactive Redis Stream outbound channel adapter to write Message payload into Redis stream.
+Outbound Channel adapter uses `ReactiveStreamOperations.add(...)` to send `Message` to stream.
+The following example shows how to use Java Configuration and Service class for Redis Stream Outbound Channel Adapter.
+
+====
+[source,java]
+----
+@Service
+public class MyService {
+
+    @Autowired
+    private FluxMessageChannel messageChannel;
+
+    public void sendMessage(Flux<Object> myFlux) {
+        messageChannel.subscribeTo(myFlux.map(myObj -> new GenericMessage<>(myObj)));
+    }
+}
+
+@Configuration
+@EnableIntegration
+public class MyConfiguration {
+
+    @Bean
+    FluxMessageChannel messageChannel() {
+        return new FluxMessageChannel();
+    }
+
+    @Bean
+    @ServiceActivator(inputChannel = "messageChannel")
+    public ReactiveRedisStreamMessageHandler reactiveValidatorMessageHandler(ReactiveRedisConnectionFactory reactiveRedisConnectionFactory) {
+        return new ReactiveRedisStreamMessageHandler(reactiveRedisConnectionFactory, "myStreamKey");
+    }
+}
+----
+====
+
 [[redis-stream-inbound]]
 === Redis Stream Inbound Channel Adapter
 
-TBD
+Spring Integration 5.4 introduced the Reactive Stream inbound channel adapter for reading messages from a Redis Stream.
+Inbound channel adapter uses `StreamReceiver.receive(...)` or `StreamReceiver.receiveAutoAck` based on
+auto acknowledgement flag to read `Message` from redis stream.
```

Comment:
Forgot to point to this one as well: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line